docs/input/docs/plugins/10-plugins.md

```diff
@@ -0,0 +1,19 @@
+Description: Tasty Plugins
+Title: Tasty Plugins
+Order: 10
+---
+
+Cause `Tasty` is very open and flexible from the ground up almost everything is based on plugins.  
```

Comment:
Cause -> Because

Reply:
Everytime my *Slang* gets through 🤔
Should add some automation that kills that on ci level